### ISSUE TITLE
Poloniex

### ISSUE THREAD

**Issue by pottertech** [ccxt/ccxt#15131]
- OS: Ubuntu 20.04
- Programming Language version: PHP
- CCXT version: 1.93.108

```
            $exchangeName = '\ccxt\\poloniex';
            $exchange = new $exchangeName(array(
                    'apiKey' => $apiKey,
                    'secret' => $secretKey,
            ));

            $params = array();
            $exchange->load_markets();
            $exchange->verbose = true;
            $balance = $exchange->fetch_balance($params);
```

``` Verbose Dump
Array
(
    [0] => fetch Request:
    [1] => poloniex
    [2] => GET
    [3] => https://api.poloniex.com/accounts/balances?accountType=SPOT
    [4] => RequestHeaders:
    [5] => Array
        (
            [Content-Type] => application/json
            [key] => TMS6HK1I-8VG0B8I2-T9XMY3NL-GAOHEXX9
            [signTimestamp] => 1664374222339
            [signature] => qwfsgTVANnJooLCCHM5+QlWp0OPl2UOTF6WWADS64qU=
        )

    [6] => RequestBody:
    [7] =>
)
Array
(
    [0] => fetch Response:
    [1] => poloniex
    [2] => GET
    [3] => https://api.poloniex.com/accounts/balances?accountType=SPOT
    [4] => 401
    [5] =>
    [6] => ResponseHeaders:
    [7] => Array
        (
            [Date] => Wed, 28 Sep 2022 14:10:22 GMT
            [Content-Length] => 0
            [Connection] => keep-alive
            [RateLimit-Filter] => updated
            [Web-Filter] => unauthenticated
            [CF-Cache-Status] => DYNAMIC
            [Server] => cloudflare
            [CF-RAY] => 751d0fe99843255a-SJC
        )

    [8] => ResponseBody:
    [9] =>
)
```

**Comment by carlosmiei** [comment]
Hello @pottertech as you can see you're getting a 401 error that means "unauthorized" so I would suggest double-checking your API keys. I've also tested on my side and worked fine:

```Php

PHP v8.1.10
CCXT v1.93.66
poloniex->fetchBalance()
Array
(
    [info] => Array
        (
            [0] => Array
                (
                    [accountId] => 76118648825274368
                    [accountType] => SPOT
                    [balances] => Array
                        (
                            [0] => Array
                                (
                                    [currencyId] => 125
                                    [currency] => LTC
                                    [available] => 0.07847347
                                    [hold] => 0.00
                                )

                            [1] => Array
                                (
                                    [currencyId] => 214
                                    [currency] => USDT
                                    [available] => 6.7976210783525
                                    [hold] => 17.09
                                )

                            [2] => Array
                                (
                                    [currencyId] => 267
                                    [currency] => ETH
                                    [available] => 0.00116007925
                                    [hold] => 0.00
                                )

                            [3] => Array
                                (
                                    [currencyId] => 697
                                    [currency] => BTTOLD
                                    [available] => 0.001888
                                    [hold] => 0.00
                                )

                            [4] => Array
                                (
                                    [currencyId] => 754
                                    [currency] => ETHW
                                    [available] => 0.00116008
                                    [hold] => 0.00
                                )
(...)
```

**Comment by pottertech** [comment]
OK, I re-entered the API info and now it worked.  Thanks a ton!
